blazor/blazor-playground/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: Overview of Blazor Playground | Syncfusion
+description: Syncfusion Blazor Playground offers a comprehensive playground within your browser, empowering you to write, edit, compile, and share your Blazor components effortlessly. Explore the capabilities of Blazor components in real-time without the need for complex setups or external tools.
+platform: Blazor
+component: Common
+documentation: ug
+---
+# Overview of Blazor Playground
+
+[Blazor Playground](https://blazorplayground.syncfusion.com/) is a web-based application designed to compile and test Blazor components. It offers a platform where developers can write, visualize, and experiment with Blazor components using C# and Razor syntax. The Playground's functionality allows users to quickly iterate and validate their code without the need for a local development environment. It serves as an efficient tool for learning Blazor, prototyping ideas, and sharing code snippets with others. The code compilation utilizes WebAssembly technology which gives a performance that closely resembles native execution.
+
+The Syncfusion Blazor Playground for Blazor application interface comprises the following sections:
+* Appbar
+* Sidebar
+* Editor
+* Result View
+* Error List
+
+## Appbar
+The appbar of Syncfusion Blazor playground allows you to access the following functionalities: 
+![Appbar](images/Appbar.png)
+## Run
+The code from the editor is compiled, and the output is displayed in the Result View.
+## Share
+you can obtain a URL link to exchange your code snippets.
+There are two ways to share the code snippet
+### Shared Link:
+A shared link is useful if you want to allow others to customize the code snippet.
```

Comment:
Expecting to add the images for every single steps.

Reply:
Added.